.pipelines/e2e.yaml

```diff
@@ -42,14 +41,21 @@ jobs:
       export PATH="/usr/local/go/bin:$PATH"
       go version
 
-      export BUILD_ID=$(Build.BuildId)
+      if [ -z "${VHD_BUILD_ID}" ]; then
```

Comment:
did you mean `-n` here?

Reply:
`-z` will also catch an empty string. I don't have a strong opinion on which one is better in this case. But it feels a slightly safer